apps/mobile-wallet/ios/Podfile.lock

```diff
@@ -1618,7 +1618,7 @@ PODS:
     - React-Core
```

Comment:
Some paddings are wrong, maybe due to rebase...

<img width="350" alt="image" src="https://github.com/user-attachments/assets/19a7df5d-0876-4aa8-8f9f-bf2cd0ea0cc6">

<img width="350" alt="image" src="https://github.com/user-attachments/assets/e651c5c6-bbcd-495a-a1ff-082da77777f5">

Reply:
ah yep, used to be correct. Will fix